SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/RegisterAutoDestroyActorsTest/RegisterAutoDestroyActorsTest.cpp

```diff
@@ -53,14 +57,17 @@ void ARegisterAutoDestroyActorsTestPart1::BeginPlay()
 		}, 5.0f);
 	}
 
-	{ // Step 3 - Destroy by second server that doesn't have authority
-		AddServerStep(TEXT("SERVER_2_RegisterAutoDestroyActors"), 2, [](ASpatialFunctionalTest* NetTest) -> bool {
+	{ // Step 3 - Destroy by all servers that have authority
+		AddServerStep(TEXT("SERVER_ALL_RegisterAutoDestroyActors"), FWorkerDefinition::ALL_WORKERS_ID, [](ASpatialFunctionalTest* NetTest) -> bool {
 			int NumCharactersFound = 0;
-			int NumCharactersExpected = NetTest->GetNumberOfServerWorkers();
+			int NumCharactersExpected = 1;
 			UWorld* World = NetTest->GetWorld();
 			for (TActorIterator<ACharacter> It(World, ACharacter::StaticClass()); It; ++It)
 			{
-				++NumCharactersFound;
+				if ((*It)->HasAuthority())
```

Comment:
I thought just `It->HasAuthority()` would work?

Reply:
Similarly below

Reply:
Updated to It->HasAuthority().